cli_li3ds/import_platform.py

```diff
@@ -85,3 +97,29 @@ def take_action(self, parsed_args):
         objs.add(pconfig)
         objs.get_or_create()
         self.log.info('Success!\n')
+
+    @staticmethod
+    def read_lidar_rigid_transform_params(filename):
+        config = configparser.ConfigParser()
+        config.read(filename)
+
+        section = 'TerraMatch corrections v2'
+        easting = config.getfloat(section, 'EastingShift')
+        northing = config.getfloat(section, 'NorthingShift')
+        elevation = config.getfloat(section, 'ElevationShift')
+        heading = config.getfloat(section, 'HeadingShift')
+        roll = config.getfloat(section, 'RollShift')
+        pitch = config.getfloat(section, 'PitchShift')
+
+        t0 = math.cos(heading * 0.5)
+        t1 = math.sin(heading * 0.5)
+        t2 = math.cos(roll * 0.5)
+        t3 = math.sin(roll * 0.5)
+        t4 = math.cos(pitch * 0.5)
+        t5 = math.sin(pitch * 0.5)
+        qw = t0 * t2 * t4 + t1 * t3 * t5
+        qx = t0 * t3 * t4 - t1 * t2 * t5
+        qy = t0 * t2 * t5 + t1 * t3 * t4
+        qz = t1 * t2 * t4 - t0 * t3 * t5
+
```

Comment:
Hum... radian conversion is definitely missing there.

Here is the code we use to compute the rotation when using a sbet to transform a local metric frame to Lambert93 (the translation being performed by proj4)
```
    double conv_meridien = 0.72537437089 * (event.m_longitude-0.0523598775598); // sin(0.811578102)=0.72537437089
    double heading = event.m_plateformHeading-event.m_wanderAngle-conv_meridien;
    double cp = cos(event.m_pitch), sp = sin(event.m_pitch);
    double cr = cos(event.m_roll), sr = sin(event.m_roll);
    double ch = cos(heading), sh = sin(heading);
    return [      cp*sh, sr*sp*sh+cr*ch, cr*sp*sh-sr*ch,
                      cp*ch, sr*sp*ch-cr*sh, cr*sp*ch+sr*sh,
                      sp   ,-sr*cp         ,-cr*cp         ];
```

but I still have to figure out an equivalent quaternion...